README.md

```diff
@@ -1,46 +1,65 @@
+- [Drug Safety Update](https://www.gov.uk/drug-safety-update)
+- [ESI Funds](https://www.gov.uk/european-structural-investment-funds)
+- [International Development Funds](https://www.gov.uk/international-development-funding)
+- [Medical Safety Alerts](https://www.gov.uk/drug-device-alerts)
+- [MAIB Reports](https://www.gov.uk/maib-reports)
+- [RAIB Reports](https://www.gov.uk/raib-reports)
+- [Vehicle Recalls and Faults Alerts](https://www.gov.uk/vehicle-recalls-faults)
+- Manuals: There is no public index page for Manuals. They can be found at `gov.uk/guidance/:manual-slug`.
+
 ## Nomenclature
 
-* Specialist Documents: Documents with metadata which are published to Finders
-* Schema: JSON file defining slug, document noun and name of Specialist Document document_types. Also has select facets and their possible values for each document_type which are displayed by the `_form.html.erb`.
-* Manual: Grouped Documents published as a number of sections inside a parent document
 
-## Current formats
+- **Format**: Category of a Document. Format names are listed in the `Live Examples` section above and include `MAIB Reports` and `CMA Cases`.
+- **Document**: Specialist Documents are created by Government editors and can be published to gov.uk. Documents differ from each other depending on their format. These differences are largely determined by what is contained in the [schema](https://github.com/alphagov/specialist-publisher-rebuild/blob/add-dfid-review-status/lib/documents/schemas/aaib_reports.json) of a format.
+- **Schema**: JSON files defining attributes for each format, including `base_path`, `document_noun` and `document_type`. It also contains the facets and their possible values for each `document_type` which are displayed by `_form.html.erb`.
+- **Manual**: Grouped Documents published as a number of sections inside a parent document.
 
-### Live
+## Technical documentation
 
-* [AAIB Reports](https://www.gov.uk/aaib-reports)
-* [CMA Cases](https://www.gov.uk/cma-cases)
-* [Countryside Stewardship Grants](https://www.gov.uk/countryside-stewardship-grants)
-* [International Development Funds](https://www.gov.uk/international-development-funding)
-* [Drug Safety Update](https://www.gov.uk/drug-safety-updates)
-* [ESI Funds](https://www.gov.uk/esi-funds)
-* [Medical Safety Alerts](https://www.gov.uk/drug-device-alerts)
-* [MAIB Reports](https://www.gov.uk/maib-reports)
-* [RAIB Reports](https://www.gov.uk/raib-reports)
-* Manuals (there's no public index page for Manuals, they can all be found at `gov.uk/guidance/:manual-slug`)
+Specialist Publisher Rebuild is a Ruby on Rails application used to create and manage documents and manuals. This application does not store documents and manuals in a database of its own. Instead it sends JSON data to the publishing-api where it is persisted in a Postgres datastore. This data is then requested from the publishing-api and displayed to the user.
```

Comment:
What do you think about mentioning Rummager / Email Alert API and Asset Manager in this bit? It might be too much detail, but could be useful to see how it all fits together.